pkg/apis/apis/v1alpha1/types_apiexport.go

```diff
@@ -69,6 +69,12 @@ type APIExport struct {
 	Status APIExportStatus `json:"status,omitempty"`
 }
 
+const (
+	// AnnotationAPIExportExtraKeyPrefix is the prefix of an annotation set on an APIExport to provide extra info. The
+	// annotation with the prefix will be synced to all the APIBindings bound to this APIExport.
```

Comment:
```suggestion
	// AnnotationAPIExportExtraKeyPrefix is the prefix of an annotation set on an APIExport to provide extra info that will be made available to all APIBindings bound to this APIExport.
	// Any annotation with this prefix will be continuously synced to all the APIBindings bound to this APIExport. If the annotation is removed from the APIExport, it will also be removed from all APIBindings bound to this APIExport.
```